client/stylesheets/_responsive.scss

```diff
@@ -359,14 +351,32 @@
 }
 
 @media (max-width: 600px) {
-    
+    .profileCardLi {
+        margin: 6px 5px !important;
+    }
 }
 
+@media (max-width: 583px) {
+      .profileCardLi {
+        width: 315px !important;
+      }
+
+      .profileCardInfo li:first-child {
+        margin-bottom: 12px !important;
+      }
+
+      .profileCardItem {
```

Comment:
Line should be indented 2 spaces, but was indented 6 spaces<br>Selector `profileCardItem` should be written in lowercase with hyphens